apps/auction/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "auction",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gear-js/api": "^0.24.2",
+    "@gear-js/react-hooks": "^0.2.16",
+    "@gear-js/ui": "^0.4.0",
+    "@mantine/form": "^4.2.12",
+    "@polkadot/api": "^8.13.1",
+    "@polkadot/extension-dapp": "^0.43.3",
+    "@polkadot/react-identicon": "^2.8.2",
+    "@polkadot/types": "^8.13.1",
+    "@polkadot/util": "^9.7.2",
+    "@polkadot/wasm-crypto": "^6.3.1",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.11.44",
+    "@types/react": "^18.0.15",
+    "@types/react-dom": "^18.0.6",
+    "clsx": "^1.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "^5.0.1",
+    "react-transition-group": "4.4.2",
+    "sass": "^1.54.0",
+    "typescript": "^4.7.4",
```

Comment:
it's dev dep, same as @types

Reply:
[source](https://stackoverflow.com/a/44872787)

Reply:
Information outdated, do not agree